### ISSUE TITLE
apiVersion: networking.k8s.io/v1beta1 kind: Ingress

### ISSUE THREAD

**Issue by BertrandGouny** [containous/traefik-migration-tool#3]
Hello,
ingress with this kind of definition are ignored :
```
apiVersion: networking.k8s.io/v1beta1
kind: Ingress
metadata:
  name: test-ingress
  annotations:
    nginx.ingress.kubernetes.io/rewrite-target: /
spec:
  rules:
  - http:
      paths:
      - path: /testpath
        backend:
          serviceName: test
          servicePort: 80
```

this example is from official documentation : 
https://kubernetes.io/docs/concepts/services-networking/ingress/

error : 
ingress.go:90: object is not an ingress ignore it: *v1beta1.Ingress

**Comment by ldez** [comment]
Hello,

I problem is fixed in v0.9.0

**Comment by BertrandGouny** [comment]
Thanks !
